<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WaveForm</class>
 <widget class="QWidget" name="WaveForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>581</width>
     <height>401</height>
    </rect>
   </property>
   <property name="title">
    <string>实时曲线显示</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QCustomPlot" name="widget_plot" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnexit">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>参数1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_pra1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>参数2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_pra2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnexit</sender>
   <signal>clicked()</signal>
   <receiver>WaveForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
